src/app/modules/submission/submission.service.ts

```diff
@@ -524,7 +546,16 @@ export const uploadAttachments = (
       })
       return new AttachmentUploadError()
     },
-  ).map(() => attachmentMetadata)
+  ).map(() => {
+    logger.info({
+      message: 'Successfully uploaded attachments to S3',
```

Comment:
does not necessarily help to identify the submission. but gives info about successful s3 uploads, similar to when submission is saved to mongodb successfully.